Cargo.toml

```diff
@@ -1,53 +1,10 @@
 [workspace]
```

Comment:
```suggestion
[workspace]
resolver = "2"
```

Reply:
This is the default for Rust 2021 https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html#details

Reply:
Did you find any docs on this? I can't find anything suggesting that this is a valid addition offhand, but it kindof makes sense? 

All of the subcrates are "edition = 2021", which implies "resolver = 2" by default. But since resolution happens globally, is this important too?

Reply:
Ah, found it! I must have skimmed the relevant part of the docs before :)

Reply:
(ah sorry @ascjones; I hadn't seen your reply)